icubam/db/store.py

```diff
@@ -544,6 +600,10 @@ def get_password_hash(self, password: str) -> str:
       "sha256", password.encode("utf8"), self._salt, 100000
     ).hex()
 
+  def make_token(self, user_id: int, icu_id: int) -> str:
+    token_hash = self.get_password_hash(json.dumps([user_id, icu_id]))
```

Comment:
Nice use of pbkdf2_hmac to generate a hash per user/icu pair. I would have probably tried random strings at insertion time in the DB, but there is indeed a risk if collision.